pyproject.toml

```diff
@@ -1,7 +1,8 @@
 [tool.poetry]
 name = "ast-comments"
-version = "1.2.0"
+version = "1.2.1"
 description = ""
+license = "MIT"
```

Comment:
Hi. Can you explain please why `license` here is important? I mean I've seen pyproject files that have it and files that don't.

Reply:
It tells pypi and all pypi/poetry.lock parsers your license, the most important piece of metadata in your project for companies with proprietary code.

At my company, we have a parser that takes the entire poetry.lock and looks through all dependencies. We and most other companies cannot legally use unlicensed or strict copyleft licensed software which is why this parser flagged your library.

Reply:
Understood. Thanks for clearing this up.